radio/src/opentx.cpp

```diff
@@ -2552,13 +2548,7 @@ void opentxInit(OPENTX_INIT_ARGS)
 
   if (g_eeGeneral.backlightMode != e_backlight_mode_off) backlightOn(); // on Tx start turn the light on
 
-  if (UNEXPECTED_SHUTDOWN()) {
-#if !defined(CPUARM)
-    // is done above on ARM
```

Comment:
FYI it was done this way (yes, difficult to understand) to save some bytes of flash on 9X. I agree on revoving this